frontend/www/js/omegaup/components/contest/NewForm.vue

```diff
@@ -320,14 +355,39 @@ export default class NewForm extends Vue {
     this.penalty = 20;
     this.penaltyType = 'none';
     this.showScoreboardAfter = true;
+    this.contestStyleDesc = T.contestNewFormTitlePlaceholderConacupStyle;
   }
 
   onSubmit() {
-    this.$emit('emit-update-contest', this);
-  }
-
-  mounted() {
-    $('.selectpicker', this.$el).selectpicker();
+    if (this.update) {
+      this.$emit('emit-update-contest', this);
+      return;
+    }
+    this.$emit('create-contest', {
+      alias: this.alias,
+      title: this.title,
+      description: this.description,
+      start_time: this.startTime
+        ? this.startTime.getTime() / 1000
+        : this.initialStartTime.getTime() / 1000,
+      finish_time: this.finishTime
+        ? this.finishTime.getTime() / 1000
+        : this.initialFinishTime.getTime() / 1000,
+      window_length:
+        this.windowLength === null || !this.windowLengthEnabled
+          ? 0
```

Comment:
esto no debería ser nulo?

```suggestion
          ? null
```

el controlador permite que esto no se envíe.

Reply:
Lo estoy intentando mandar `null`, pero el controlador recibe una cadena = `null` y hace que no funcione la validación de entero

Reply:
oh! https://github.com/omegaup/omegaup/pull/4018 debería arreglar eso.

Reply:
Gracias, ya se guarda correctamente.